### PR TITLE
implement RNNAttention

### PR DESCRIPTION
**Pull request by yangtzech**
@oguiza , please take a look at this when you have time.

### CODE REVIEW

File: nbs/075_models.RNNAttention.ipynb [hunk 1]
```diff
@@ -0,0 +1,540 @@
+{
```
Comment (oguiza) [inline nbs/075_models.RNNAttention.ipynb:1]: <p>You should replace by name by yours. It's your implementation. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/timeseriesAI/tsai/pull/668/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (oguiza) [inline nbs/075_models.RNNAttention.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #9.            Args:
</pre><p>You should replace the line that explain what this model is. It's certainly not TST. I don't know if you are basing this model on any particular paper, or it's just an implementation you have created based on your experience. Can you please comment on it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/timeseriesAI/tsai/pull/668/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

**Comment by oguiza** [comment]
Hi @yangtzech ,
Thanks for creating this PR. I think it's a great addition to tsai. 
I have a few comments: 
1. There're a couple of other files that need to be modified to let the new models be constructed within the ts_learner, but I can take care of that once this PR is approved.
2. I'm getting an error message with the PR: "Error: Notebooks and library are not in sync.  Please run nbdev_export.". You should run 2 things before resubmitting the PR: nbdev_install_hooks and nbdev_prepare. 
Look forward to seeing the new model added to tsai.

File: nbs/075_models.RNNAttention.ipynb (hunk 1 above)
Comment (yangtzech) [reply nbs/075_models.RNNAttention.ipynb:1]: <p>Ok, I'll edit it.</p>

Comment (yangtzech) [reply nbs/075_models.RNNAttention.ipynb:1]: Sorry for the mistake. The model is apparently based on the TST model you implement. I created it for my own need. And I also see someone filed issue #595. Therefore I made this PR. I'll edit it.

**Comment by yangtzech** [comment]
I think I'll close this PR before messing up the repo.

**Pull request closed without merging**